Quotient/connection.cpp

```diff
@@ -554,8 +750,18 @@ void Connection::Private::consumeRoomData(SyncDataList&& roomDataList,
             // Update rooms one by one, giving time to update the UI.
             QMetaObject::invokeMethod(
                 r,
-                [r, rd = std::move(roomData), fromCache] () mutable {
+                [r, rd = std::move(roomData), fromCache, this] () mutable {
                     r->updateData(std::move(rd), fromCache);
+                    if (r->usesEncryption()) {
+                        rust::Vec<rust::String> userIds;
+                        const auto ids = r->memberIds();
+                        for (const auto &id : ids) {
+                            userIds.push_back(stringToRust(id));
+                        }
+                        if (cryptoMachine) {
+                            (*cryptoMachine)->update_tracked_users(userIds);
+                        }
+                    }
```

Comment:
```suggestion
                    if (cryptoMachine && r->usesEncryption()) {
                        auto userIds = rangeTo<rust::Vec<rust::String>>(std::ranges::transform_view(r->memberIds(), &stringToRust));
                        (*cryptoMachine)->update_tracked_users(userIds);
                    }
```
Or something like this. And maybe `userIds` can even be `const`. And maybe we could do with `stringsToRust()` because it's not the only place this logic is needed.